docs/interactive/launchbuttons.ipynb

```diff
@@ -19,7 +19,24 @@
     "  url                       : https://github.com/yourusername/yourbookrepo  # Online location of your book\n",
     "  path_to_book              : path/to/book  # Optional path to your book, relative to the repository root\n",
     "  branch                    : master  # Which branch of the repository should be used when creating links (optional)\n",
-    "```"
+    "```\n",
+    "The launch button will use [nbgitpuller](https://github.com/jupyterhub/nbgitpuller) to pull your\n",
+    "repository to the chosen Binder, Google Colab, or JupyterHub location.  The `repository:`\n",
+    "configuration line uses `url`, `path_to_book`, and `branch` to find your notebook files _both MyST\n",
```

Comment:
I think we could just leave it as "your notebook files" - I suspect that it may confuse people if we talk too much about MyST Markdown Notebooks vs. ipynb files...perhaps there should be an explicit section that describes differences in their behavior?

Reply:
That might be better to leave it as "notebook files", it covers the ipynb and myst-nb files. 

I agree, it will help to have a separate document that addresses how Jupyter Book deals with:
* markdown
* myst markdown ([already covered](https://jupyterbook.org/content/myst.html))
* ipynb

Playing around with jupytext, I found that most markdown (myst-nb and common markdown) will build HTML, but any myst-directives show up as raw code in Binder/JupyterHub. 

This may change, but for now I found the most portable solutions are *.ipynb and *.md using common markdown and jupytext-header, where *.md headers are:

```
---
jupyter:
  jupytext:
    text_representation:
      extension: .md
      format_name: markdown
      format_version: '1.2'
      jupytext_version: 1.6.0
  kernelspec:
    display_name: Python 3
    language: python
    name: python3
---
```

and code cells are shown with
````
```python tags=["hide-cell"]
import numpy as np
import matplotlib.pyplot as plt
plt.style.use('fivethirtyeight')
```
````